clair-scanner/scan.sh

```diff
@@ -48,4 +48,8 @@ else
     scan "<< parameters.image >>"
 fi
 
-exit $EXIT_STATUS
+if [ "<< fail_on_discovered_vulnerabilities >>" = "false"]
```

Comment:
Not completely sure how the boolean parameter type plays here.

Reply:
CircleCI needs `parameters.` before the parameter name. 
Bash needs a space before the `]`, and `==` for the comparison.

```suggestion
if [ "<< parameters.fail_on_discovered_vulnerabilities >>" == "false" ]; then
```